images/app/helpers/refinery/admin/images_helper.rb

```diff
@@ -19,6 +19,12 @@ def thumbnail_urls(image)
 
         thumbnail_urls
       end
+
+      # We show the title from the next available locale
+      # if there is no title for the current locale
```

Comment:
We do that on the backend because otherwise it is simply blank, but we don't want to display wrong content on the frontend.  Maybe we should write the pages one better too.